<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logon</class>
 <widget class="QDialog" name="logon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登陆</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#logon{background-image: url(:/imgs/imgs/LogonImg.png);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usrLineEdit">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>143</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(119, 226, 137);
border:none;
border-radius:3px;
background-color:rgba(255, 255, 255, 180);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>170</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLineEdit">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>180</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(119, 226, 137);
border:none;
border-radius:3px;
background-color:rgba(255, 255, 255, 180)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logonButton">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>230</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
color:white;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
